tests/Feature/DockerTagsTest.php

```diff
@@ -49,7 +49,7 @@ function it_sorts_the_versions_naturally()
         $tags = collect($dockerTags->getTags());
 
         $this->assertEquals('latest', $tags->shift());
-        $this->assertEquals('17.2', $tags->shift());
+        $this->assertEquals('17.4', $tags->shift());
```

Comment:
![image](https://github.com/user-attachments/assets/e01bc0e3-ef53-49ea-8eff-98224d66dc82)

https://www.postgresql.org/about/news/postgresql-174-168-1512-1417-and-1320-released-3018/